app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from .errors import (NotWearingMaskError, VaccineError)
```

Comment:
Use absolute imports instead of relative imports as per the checklist. Change `from .errors import ...` to `from app.errors import ...`.